packages/client/mutations/UpdateFeatureFlagMutation.ts

```diff
@@ -0,0 +1,41 @@
+import graphql from 'babel-plugin-relay/macro'
+import {commitMutation} from 'react-relay'
+import {StandardMutation} from '../types/relayMutations'
+import {UpdateFeatureFlagMutation as TUpdateFeatureFlagMutation} from '../__generated__/UpdateFeatureFlagMutation.graphql'
+
+graphql`
+  fragment UpdateFeatureFlagMutation_notification on UpdateFeatureFlagPayload {
+    user {
+      id
+      # add flag here
```

Comment:
I'm not sure what that comment means? Do we use this mutation on client? Maybe remove it completely, or do we plan to use it in future?

Reply:
I took this from the `AddFeatureFlagMutation` - I think if we want to notify users, we can include the flag here

Reply:
Yeah, I mean, I don't see we use `AddFeatureFlagMutation` or `UpdateFeatureFlagMutation` on client. We export it but don't import

Reply:
I think we used to let users add themselves to a feature using a cheat code, like if they typed in "Spotlight", they'd get access to it.

I could see that happening again in the future. For instance, if they could request access to the Insights beta feature and we'd automatically turn it on. So I think it's safe to keep it